README.md

```diff
@@ -10,6 +10,7 @@
 [![Build Status - Cirrus][]][Build status]
 [![Discord badge][]][Discord instructions]
 [![Twitter handle][]][Twitter badge]
+[![codecov](https://codecov.io/gh/flutter/flutter/branch/master/graph/badge.svg?token=11yDrJU2M2)](https://codecov.io/gh/flutter/flutter)
```

Comment:
So it looks like this is only coverage for Flutter tools. Do we still want to show this badge? And if so, we should probably clarify.

Reply:
https://app.codecov.io/gh/flutter/flutter/tree/master/packages

Reply:
Actually, we shouldn't land this, we haven't uploaded these reports in years

Reply:
https://app.codecov.io/gh/flutter/flutter/commits

Reply:
Just fixed that yesterday: https://github.com/flutter/flutter/actions/workflows/coverage.yml we will be generating per commit reports starting in a few mins from now.

Reply:
Awesome, I see this updated now, LGTM!